gravitee-am-ui/src/styles.scss

```diff
@@ -1,7 +1,7 @@
 /* You can add global styles to this file, and also import other style files */
 @use 'sass:map';
 @use '@angular/material' as mat;
-@use '@gravitee/ui-particles-angular' as gio;
+@use '../node_modules/@gravitee/ui-particles-angular' as gio;
```

Comment:
is it normal to have this change ?

Reply:
It's fixed with new configuration https://github.com/gravitee-io/gravitee-access-management/pull/2832/commits/e0dd7b4f2d17103d1e8c5ccc1a8a413b11d4e173#diff-8ba6ed89361dfb033be734a07d69340f45aa031f255cc8de7461608262bb4690R68-R72